data/transactions/logic/fields.go

```diff
@@ -1180,6 +1183,7 @@ func (fs assetHoldingFieldSpec) Note() string {
 var assetHoldingFieldSpecs = [...]assetHoldingFieldSpec{
 	{AssetBalance, StackUint64, 2, "Amount of the asset unit held by this account"},
 	{AssetFrozen, StackBoolean, 2, "Is the asset frozen or not"},
+	{AccountFrozen, StackBoolean, 12, "Is the account frozen or not"},
```

Comment:
I see where you're going now, but take my earlier confusion as indicative of eventual user confusion. "AssetFrozen" and "AccountFrozen"? Huh?